### ISSUE TITLE
PolyPen: Unhandled Exception Caught!

### ISSUE THREAD

**Issue by ColdNightHour** [CGCookie/retopoflow#613]
Please tell us what you were trying to do, what you expected RetopoFlow to do, and what actually happened.
Provide as much information as you can so that we can reproduce the problem and fix it.
Screenshots and .blend files are very helpful.
Also, change the title of this bug report to something descriptive and helpful.
Thank you!

-------------------------------------
Was trying to use the PolyPen to add a vertex to an edge
```
Environment:

- RetopoFlow: 2.0.0
- Blender: 2.79.0 master 2018-03-22
- Platform: Windows, 10, 10.0.17134, AMD64, Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
- GPU: NVIDIA Corporation, GeForce GTX 1050 Ti with Max-Q Design/PCIe/SSE2, 4.6.0 NVIDIA 398.36, 4.60 NVIDIA
- Timestamp: 2018-08-31 11:29:45.900061


Error Hash: 9a42177a85e4c84a725eab0bc42f53aa


Trace:

- EXCEPTION (<class 'ValueError'>): edges.new(): this edge exists
- .../rfmode\rfmesh.py
- 000 1331:new_edge() bme = self.bme.edges.new(verts)
- .../rfmode\rfcontext_target.py
- 001 0396:new_edge() return self.rftarget.new_edge(verts)
- .../rfmode\rftool_polypen.py
- 002 0282:modal_insert() bme = self.rfcontext.new_edge((bmv0, bmv1))
- .../rfmode\rftool.py
- 003 0076:wrapper() ret = fn(*args, **kwargs)
- 004 0110:modal() nmode = self.FSM[self.mode]()
- 005 0114:modal() raise e     # passing on the exception to RFContext
- .../rfmode\rfcontext.py
- 006 0556:modal_main() self.tool.modal()
- 007 0469:modal() nmode = self.FSM[self.mode]()
```

**Comment by vxlcoder** [comment]
Thanks for the report, @ColdNightHour.  Could you post a screenshot or your .blend file?  Can you reproduce this bug on a different mesh, like the default cube?  Were you continuing work on a retopo target mesh, or did you start a new target mesh?

**Comment by vxlcoder** [comment]
This is the same issue as #603, and it should now be fixed in the latest version of `release/2.0.1` branch. 
 If you know how, feel free to test that version, otherwise it will be officially released soon.

I'm closing this issue, but reopen if you continue to have problems.

Thanks for reporting this issue!
